public/reportissue.php

```diff
@@ -181,43 +222,40 @@ function displayCore() {
             <tr>
                 <td><label for="description">Description</label></td>
                 <td colspan="2">
-                    <textarea class="w-full forum" name="description" id="description"
-                              style="height:160px" rows="5" cols="61" placeholder="Describe your issue here..."
-                              required data-bind="textInput: description"><?= old('description') ?></textarea>
-                    <p data-bind="visible: descriptionIsNetworkProblem">Please do not use this tool for network issues. See <a href='https://docs.retroachievements.org/FAQ/#how-can-i-get-credit-for-an-achievement-i-earned-but-wasnt-awarded'>here</a> for instructions on how to request a manual unlock.</p>
-                    <p data-bind="visible: descriptionIsUnhelpful">Please be more specific with your issue&mdash;such as by adding specific reproduction steps or what you did before encountering it&mdash;instead of simply stating that it doesn't work. The more specific, the better.</p>
+                    <textarea
+                        class="w-full forum"
+                        name="description"
+                        id="description"
+                        style="height:160px"
+                        rows="5"
+                        cols="61"
+                        placeholder="Describe your issue here..."
+                        required
+                        x-model="description"
+                    >
+                        <?= old('description') ?>
+                    </textarea>
+
+                    <p x-show="descriptionIsNetworkProblem">
+                        Please do not use this tool for network issues.
+                        See <a href='https://docs.retroachievements.org/FAQ/#how-can-i-get-credit-for-an-achievement-i-earned-but-wasnt-awarded'>here</a>
+                        for instructions on how to request a manual unlock.
+                    </p>
+                    <p x-show="descriptionIsUnhelpful">
+                        Please be more specific with your issue&mdash;such as by adding specific reproduction steps or what you 
+                        did before encountering it&mdash;instead of simply stating that it doesn't work. The more specific, the better.
+                    </p>
```

Comment:
This text is briefly visible when loading (or reloading) the page.
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/c2cf8c3a-873e-4e71-b42d-9c6d1b91f7f5)

Reply:
Thanks, I didn't see this with a local dev build. Resolved in latest with the addition of an `x-cloak` directive.